src/CommitmentsV2/SFA.DAS.CommitmentsV2.Shared/Extensions/DecimalExtensions.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace SFA.DAS.CommitmentsV2.Shared.Extensions
+{
+    public static class DecimalExtensions
+    {
+        public static string ToGdsCostFormat(this decimal value)
+        {
+            return $"£{value:n0}";
+        }
+    }
```

Comment:
I can't appear to find any tests for this code?

Reply:
Added new tests